students/tri_nguyen/lesson03/list_lab.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python3
+
+# ------------------------------------------- #
+# Student:   Tri Nguyen
+# Instructor: Natasha Aleksandrova
+# Date:  09-Feb-2018
+# ------------------------------------------- #
+
+# ------------------------------------------- #
+# Series 1:
+# - Create a list that contains 'Apples', 'Pears',
+#   'Oranges' and 'Peaches.
+# - Display the list.
+# - Ask the user for another fruit and add it
+#   to the end of the list.
+# - Display the list.
+# - Ask the user for a number and display the number
+#   back to the user and the fruit corresponding
+#   to that number.
+# - Add another fruit to the beginning of the list
+#   using '+' and display the list.
+# - Add another fruit to the beginning of the list
+# - using insert() and display the list.
+# - Display all the fruits that begins with 'P',
+#   using a for loop.
+
+
+fruit_list = ['Apples', 'Pears', 'Oranges', 'Peaches']
+
+print('--Series 1 of the exercise--'.upper()+'\n')
+
+print('Below is the current list of fruits.')
+for idx, fruit in enumerate(fruit_list):
+    print(idx + 1, '-', fruit)
+
+print()
+user_input = input('Please add another fruit: ')
+fruit_list.append(user_input.title())
```

Comment:
for this exercise it is a good idea to not modify original list since all series/tasks rely on the original content of it.

you could have defined a function for each series and within function scope have the original `fruit_list` or define `fruit_list` at the top of the file and make a copy within each function.